hud-data/sigma-wolf-hud.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "sigma wolf hud",
+  "author": "zxxtlz",
+  "releaseDate": "2024/02/02",
+  "flags": [
+    "fonts",
+    "menus",
+    "scoreboards",
+    "minmode",
+    "materials",
+    "customization"
+  ],
+  "tags": [
+    "minimal",
+    "meme"
+  ],
+  "repo": "https://github.com/zxxtlz/sigmawolfhud",
+  "hash": "4b0b6781dbd9dc148348cc46477d87a81787751e",
+  "prerelease": false,
+  "resources": [
+    "sigma"
+  ],
+  "social": {
+    "issues": "https://github.com/zxxtlz/sigmawolfhud/issues",
+    "steam_profile": "https://steamcommunity.com/id/metalcock",
+    "discord": "zxxtlz",
```

Comment:
If you want this to be your discord profile instead of a discord server, i believe you need to add `#0` to the end (@mastercoms may need to verify)